styles.css

```diff
@@ -7,4 +7,105 @@ grey - #95a5a6;
 red - #ee5253
 
 */
+/* add important */
 
+ 
+.hidden.pot-img, p.hidden {
+  display: none;
+}
+
+
+body {
+  background-image: url("./assets/burger-friends.jpg");
+  background-size: cover;
+  letter-spacing: -1px;
+  font-family: 'Roboto', sans-serif;
+  margin: 0px;
+  padding: 0px;
+}
+/* -------Top Bar--------- */
+
+header {
+  background-color: #0fb9b1;
+  width: 100%;
+  padding: 2px 20px;
+}
+
+header h1 {
+  font-size: 23px;
+}
+
+
+/* ---------Dish selection box------- */
+
+/* need to get the asterisk up higher */
```

Comment:
good place to play with this rule in dev tools to see if you can get it to move around

Reply:
I have been using the Dev tools way more after you showed me. Will definitely try that